pyproject.toml

```diff
@@ -69,7 +76,7 @@ convention = "google"
 [tool.ruff.lint.per-file-ignores]
 "tests/**/*.py" = []
 
-[too.ruff.format]
+[tool.ruff.format]
```

Comment:
Funny, although would put this in the same separate PR updating dev tooling as mentioned above